source/ocsp-support/ocsp-support.rst

```diff
@@ -297,10 +298,9 @@ Suggested OCSP Behavior
 For drivers with finer-grain control over their OCSP behavior, the
 suggested OCSP behavior was chosen as a balance between security and
 availability, erring on availability while minimizing network round
-trips. Therefore, in the `Suggested Stapled OCSP
-Behavior <#suggested-ocsp-behavior>`__ and `Suggested Non-Stapled OCSP
-Behavior <#_win0ckq3thfv>`__ sections, drivers are advised not to reach
-out to OCSP endpoints and CRL distribution points to minimize network
+trips. Therefore, in the `Suggested OCSP Behavior <#suggested-ocsp-behavior>`__
+section, drivers are advised not to reach out to OCSP endpoints and CRL
```

Comment:
I think we never want a driver to explicitly reach out to a CRL distribution point (though some TLS libraries may do that automagically). This is an OCSP spec, not a generalized rev checking spec.

Reply:
Are you proposing a larger change to the `Suggested OCSP Behavior` checking to remove any mention of CRL distribution points?

Reply:
No. This wording just makes it sounds like drivers must talk to CRL distribution points as part of point 7. That's not the intent. I think you need to treat these things separately. Drivers should not reach out to CRL distribution points at all. Drivers should not reach out to OCSP endpoints for intermediates.

Reply:
Changed this to not mention CRL distribution points.